app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -172,24 +170,15 @@ class _RoomProfilePageState extends ConsumerState<RoomProfilePage> {
         // Invite
         myMembership.when(
           data: (membership) {
+            if (membership == null) return const SizedBox();
             return _actionItem(
               context: context,
               iconData: Atlas.user_plus_thin,
               actionName: L10n.of(context).invite,
-              actionItemColor: membership!.canString('CanInvite')
```

Comment:
There should be bang operator here, to have it go over ```onError()``` state in case of error...